modules/ui/maproulette_details.js

```diff
@@ -1,79 +1,131 @@
+import { select as d3_select } from 'd3-selection';
+
+import { utilHighlightEntities } from '../util/index.js';
+
 export function uiMapRouletteDetails(context) {
   const l10n = context.systems.l10n;
   const maproulette = context.services.maproulette;
 
   let _qaItem;
 
 
+  function parseShortCodes(text) {
```

Comment:
the method name could be made more meaningful- we're not just parsing the codes in this method, we're actively generating content to replace them/ transforming the text. 

A comment describing the expectations of the incoming content text (heck, even with a comment showing a full example like what @bhousel often does) would also be helpful.